src/GraphQLSchemaLinter.php

```diff
@@ -79,7 +79,12 @@ public function getVersion() {
     if (!empty($this->dependencyVersions)) {
 
       foreach ($this->dependencyVersions as $name => $required) {
-        list($err, $dep_version, $stderr) = exec_manual('cd .arcanist 2>&1 >/dev/null && npm list --depth=0 2>/dev/null | grep "%C" | cut -d "@" -f 3 | tr -d "[:space:]"', $name);
+        $bin_root = dirname($this->getDefaultBinary());
+        list($stdout) = newv('ExecFuture', array('npm list -s --depth=0 --json || true'))
+          ->setCWD($bin_root)
+          ->resolvex();
```

Comment:
One more thought:

```php
list ($err, $stdout, $stderr) = newv('ExecFuture', array('npm list -s --depth=0 --json'))
    ->setCWD($bin_root)
    ->resolve();
```

Using `resolve` will return all of the execution details and _won't_ assume the command returned a non-zero result.

Then you can parse the JSON from `$stderr`.